terraform/modules/paas/main.tf

```diff
@@ -0,0 +1,44 @@
+resource cloudfoundry_service_instance postgres_instance {
+  name         = local.postgres_service_name
+  space        = data.cloudfoundry_space.space.id
+  service_plan = data.cloudfoundry_service.postgres.service_plans["${var.postgres_service_plan}"]
+  json_params  = "{\"enable_extensions\": [\"pgcrypto\", \"fuzzystrmatch\", \"plpgsql\"]}"
```

Comment:
Can we confirm the required extensions with @misaka ?

Reply:
I don't believe we need any of these, yet. Happy to leave them out until we learn/decide otherwise.